### DISCUSSION TITLE
Copilot autocompletion with v14

### DISCUSSION THREAD

**Discussion by zivkovicn** [LazyVim/LazyVim#5041] [Q&A] [answered]
Hello, 

Is it possible to do smth around this (screenshots)
![Screenshot 2024-12-13 at 7 47 59 PM](https://github.com/user-attachments/assets/bfe041b2-274c-40a0-a507-72d41655fbba)
another example
![Screenshot 2024-12-13 at 8 02 45 PM](https://github.com/user-attachments/assets/c8a4f1e9-04fa-499b-b758-f428a7d838ae)

This is (blink) autocompletion with copilot suggestion in the background, seems that it overlaps to much?

If important, I'm using https://github.com/neanias/everforest-nvim theme.

Thanks

**Comment by folke** [comment]
That's because you're overriding LazyVim's config for copilot.
Remove all your customizations.

And also check the docs for the `ai_cmp` setting

**Reply by zivkovicn** [reply, marked as answer]
@folke thanks, I managed to overcome the issue by disabling `ghost_text`

```
return {
  "saghen/blink.cmp",
  opts = {
    completion = {
      ghost_text = {
        enabled = false,
      },
    },
  },
}
```
now everything is good!
![Screenshot 2024-12-13 at 8 06 11 PM](https://github.com/user-attachments/assets/f0121d07-0f5a-4ce8-bc94-813e2a0c10e4)
